gatsby_fp/gatsby-browser.js

```diff
@@ -1,8 +1,3 @@
-/**
- * Implement Gatsby's Browser APIs in this file.
```

Comment:
Are these comments better off staying here? Not sure what's more typical.

Reply:
I decided to remove them, figured eventually we would add our own comments... we are only using it to import the style sheets so I thought the comments were a bit misleading